src/app/containers/Include/index.amp.test.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { act } from 'react-dom/test-utils';
+import AmpIncludeContainer from './index.amp';
+
+const validIdt2Props = {
+  html: `<div>Hello</div>`,
+  idt2: {
+    altText: 'some alt text',
+    dimensions: {
+      small: {
+        href: 'small.jpeg/150',
```

Comment:
```suggestion
        href: 'https://www.example.com/idt2/cb1a5166-cfbb-4520-bdac-6159299acff6/image/150',
```
- We can use a full domain here to ensure the logic to extract the image relative path from the full path is tested (I've not used smallprox as it ideally should not be publically visible as discussed above)
- We may as well use a representative path as we would expect from Ares